modules/developer_manual/pages/core/unit-testing.adoc

```diff
@@ -78,17 +80,19 @@ available.
 
 Testing apps
 
-* To run test for a specific app with the provided PHPUnit version,
-change into `<webroot>/apps/<appnname>/<testfolder>` and call
-`<webroot>/lib/composer/phpunit/phpunit/phpunit` plus optional
-parameters when needed.
+* To run the tests for a specific app with the provided PHPUnit version,
+change into `<webroot>/apps/<appname>` and
+
+....
+make test-php-unit
```

Comment:
The command in 10.1 is `test-php` - the `test-php-unit` changes should not be backported.

Reply:
will fix